docs/getting-started/install.mdx

```diff
@@ -0,0 +1,63 @@
+# Install
+
+Welcome to Widgetbook! This guide will help you get started with integrating Widgetbook into your Flutter projects and streamline your widget management process.
+
+Make sure to depend on the Widgetbook packages.
+
+There are two approaches to configure Widgetbook:
+
+1- **Manual Approach**: This approach involves using the [widgetbook package](https://pub.dev/packages/widgetbook) and manually maintaining the widget structure in Widgetbook to keep it in sync with your application.
```

Comment:
Markdown enumeration with `1.` instead of `1-`

Reply:
fixed